qiskit/compiler/assemble.py

```diff
@@ -343,7 +345,16 @@ def _parse_circuit_args(parameter_binds, backend, parametric_pulses, **run_confi
                                                        [])
     if parametric_pulses:
         run_config_dict['parametric_pulses'] = parametric_pulses
-    run_config = RunConfig(**{k: v for k, v in run_config_dict.items() if v is not None})
+
+    if meas_level:
+        run_config_dict['meas_level'] = meas_level
+        # only enable `meas_return` if `meas_level` isn't classified
```

Comment:
This comment confuses me... Should it be `# add meas_return option if meas_level is "kerneled"`?

Reply:
Oh, I see. Meas level could also be 0, so `isn't classified` is not the same as `is kerneled`, and I think you mean the second one

Reply:
Unless the below line should be ` != MeasLevel.CLASSIFIED`

Reply:
That's true, though right now there's no level 0 so they're effectively the same. I've updated the logic.

Reply:
That's true. Still, it is nice if we can write our code now such that changes we make in the future can't easily break it. Aka it's forward compatible :) Also, there could be non-IBMQ systems using Qiskit which do have measurement level 0. This will work for those systems.